packages/core/src/connectors/index.ts

```diff
@@ -108,11 +109,24 @@ export const getConnectorInstanceByType = async <T extends ConnectorInstance>(
 
 export const initConnectors = async () => {
   const connectors = await findAllConnectors();
-  const existingConnectors = new Map(connectors.map((connector) => [connector.id, connector]));
+  const existingConnectors = new Map(
+    connectors.map((connector) => [[connector.name, connector.platform].join(' '), connector])
+  );
   const newConnectors = allConnectors.filter(
-    ({ metadata: { id, type } }) => existingConnectors.get(id)?.type !== type
+    ({ metadata: { id: name, platform } }) =>
```

Comment:
should we align the property naming of id and name? As you use `connector.name` in DB but `metadata.id `in connector instances.